pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[project]
```

Comment:
Here are a few more entries that make sense here:
```
license = {file = "LICENSE"}
classifiers = [
    "Programming Language :: Python",
    "Programming Language :: Python :: 3",
    "Programming Language :: Python :: 3.9",
    "Programming Language :: Python :: 3.10",
    "Programming Language :: Python :: 3.11",
    "Programming Language :: Python :: 3.12",
    "Programming Language :: Python :: 3.13",
    "Operating System :: OS Independent",
    "Development Status :: 5 - Production/Stable",
    "Environment :: Console",
    "Intended Audience :: Developers",
    "Natural Language :: English",
    "Topic :: Software Development :: Testing",
    "Typing :: Typed",
]
```

Reply:
Not 100% sure we need to go crazy on classifiers given that this isn't going to be published anywhere.

Reply:
You never know how metadata is going to end up being used…